### PR TITLE
Add appdev tooling one page summary

### LINKED ISSUE #7279: sql: create one summary page for appdev tooling
@BramGruneir points out that our SEs and customers would benefit from one page that highlights all supported tools and as of which versions. We have a much longer-term project to document all of these tools outlined in this [meta issue](https://github.com/cockroachdb/docs/issues/6866), but, until its completed, we could fast track a summary page that lists the tool and the version.

### PR DESCRIPTION
**Pull request by vy-ton**
Fixes #7279

* Removed comprehensive/partial support classification in favor of full/beta everywhere

### CODE REVIEW

File: _includes/v20.1/misc/tooling.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+## Support
```
Comment (vy-ton) [inline _includes/v20.1/misc/tooling.md:1]: @ericharmeling @awoods187 @rafiss Would like your thoughts on this wording.

I've been working on an [internal defintion](https://docs.google.com/document/d/16jDV4BRkTMHNhN1O7NlCVaY4SFDU8WT6juELV3CzTZg/edit?usp=sharing) of full vs beta. In the meantime, this is a proposed wording for this page based on some [research](https://docs.google.com/document/d/16jDV4BRkTMHNhN1O7NlCVaY4SFDU8WT6juELV3CzTZg/edit#heading=h.66ayzme2k264)

File: _includes/v20.1/misc/tooling.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+We’ve partnered with open-source projects, vendors, and individuals to offer the following levels of support with third-party tools. If you encounter problems, please [open an issue](https://github.com/cockroachdb/cockroach/issues/new) with details to help us make progress toward better support.
```
Comment (vy-ton) [inline _includes/v20.1/misc/tooling.md:3]: I know for certain tools we point to their repos for issue tracking (Ex. [Flyway](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/9813de6273efeec08b2336240095608ecad46999/v20.1/flyway.html#report-issues-with-flyway-and-cockroachdb)). I think that is too noisy here so left the general CockroachDB issue tracker.

**Comment by vy-ton** [comment]
@ericharmeling addressed your comments, ready for another look

**Comment by awoods187** [comment]
___
*[_includes/v20.1/misc/tooling.md, line 1 at r1](https://reviewable.io/reviews/cockroachdb/docs/7393#-M87NHpEvn13If8QzO5v-r1-1:-M8Qv6Nc0-vzXvMwwUse:b-okk17c) ([raw file](https://github.com/cockroachdb/docs/blob/413db61eab2ff570746b35d5667e04d055bf9c3c/_includes/v20.1/misc/tooling.md#L1)):*
<details><summary><i>Previously, ericharmeling (Eric Harmeling) wrote…</i></summary><blockquote>

I like it!
</blockquote></details>

i like it

**Comment by jseldess** [comment]
This is a really nice page, @vy-ton!

**Comment by jseldess** [comment]
___
*[_includes/v20.1/misc/tooling.md, line 51 at r3](https://reviewable.io/reviews/cockroachdb/docs/7393#-M8SITvo-uGxhhs92ByF:-M8SITvo-uGxhhs92ByG:b-za27g3) ([raw file](https://github.com/cockroachdb/docs/blob/72ba27ae6cc2b8305ad4fd49fe86c47ff33fb68c/_includes/v20.1/misc/tooling.md#L51)):*
> ```Markdown
> ## Schema migration tools
> 
> | Tool | Recommended Version | Support level |
> ```

nit: Make `version` lowercase.

**Pull request merged**